synapse/lib/base.py

```diff
@@ -112,6 +112,12 @@ def __init__(self):
             self.tid = s_threads.iden()
             self.call_stack = traceback.format_stack()  # For cleanup debugging
 
+        if object.__getattribute__(self, 'anitted') is True:
```

Comment:
why not `getattr(self, 'anitted')`?

Reply:
this bypasses `__getattr__` on Telepath proxy classes.